frontend/src/libs/components/switch/switch.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  type Control,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
+
+import { useFormController } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties<T extends FieldValues> = {
+  control: Control<T, null>;
+  label?: string;
+  name: FieldPath<T>;
+};
+
+const Switch = <T extends FieldValues>({
+  control,
+  name,
+  label,
+}: Properties<T>): JSX.Element => {
+  const { field } = useFormController({ name, control });
+
+  return (
+    <label className={styles['container']} htmlFor={name}>
```

Comment:
we don't need htmlFor and id property for input when we nest in label

Reply:
right) Will be changed